Simplified/AxisService/AxisService.swift

```diff
@@ -0,0 +1,39 @@
+//
+//  AxisService.swift
+//  Simplified
+//
+//  Created by Raman Singh on 2021-04-01.
+//  Copyright © 2021 NYPL Labs. All rights reserved.
+//
+
+import Foundation
+
+@objc
+protocol BookDownloadBroadcasting {
+  func failDownloadWithAlert(forBook book: NYPLBook)
+  func replaceBook(_ book: NYPLBook,
+                   withFileAtURL sourceLocation: URL,
+                   forDownloadTask downloadtask: URLSessionDownloadTask) -> Bool
+}
+
+@objc
+class AxisService: NSObject {
```

Comment:
do we have a requirement to be able to build the app without Axis DRM? I know we have a target with no DRMs at all, but wondering if we need to be able to turn on/off each one. Currently that's what we're doing for Adobe and LCP, so probably might make sense to do the same here.

Reply:
Great catch. Fixed.